Home_work_3/task_3.py

```diff
@@ -0,0 +1,8 @@
+friends = ['John', 'Marta', 'James']
+enemies = ['John', 'Johnatan', 'Artur']
+
+for friend in friends:
+    if friend in enemies:
+        print(f'{friend}, we are not friends anymore')
+    else:
+        print(f'{friend}, we are the best friends')
```

Comment:
in this implementation, James will be printed to the console. But we do not need this. We do nthg with James